sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1315,6 +1315,10 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
     return getInfo(param_value_size, param_value, param_value_size_ret,
                    capabilities);
   }
+  case PI_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES:
+  case PI_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES:
+    // There is no way to query this in the backend
+    return PI_ERROR_INVALID_ARG_VALUE;
```

Comment:
How is this error reported to the end user? Should we better use set a plugin specific error and have SYCL RT use piPluginGetLastError to retrieve/report it?

Reply:
@jchlanda @npmiller could you please take a look at the changes I did in HIP/CUDA to address this comment?